test/textlintrc-test.js

```diff
@@ -82,6 +82,61 @@ describe(".textlinrc test", function() {
                 assert(result.messages[1].column === 8);
             });
         });
+        it("should not visit Link, Image, BlockQuote, Emphasis by default", async function() {
```

Comment:
`test/textlintrc-test.js` is legacy style test.
It can be moved to `test/prh-rule-tester-test.js`.

Reply:
I moved legacy style tests to test/prh-rule-tester-test.js.